base_report_to_printer/models/printing_printer.py

```diff
@@ -94,7 +96,14 @@ def print_document(self, report, content, format, copies=1):
             os.write(fd, content)
         finally:
             os.close(fd)
-
+        if copies == 1:
+            # If number of copies is not indicated by argument, check context
+            # or report definition
+            copies = (
+                self.env.context.get('report_copies') or
+                (report and report.report_copies) or
```

Comment:
As commented in https://github.com/OCA/report-print-send/pull/132/files#r180006117, the `report` variable is not an ORM object, but a string with the report name.

This must be this way because [this method is called publicly from the XMLRPC interface](https://github.com/OCA/report-print-send/blob/2bae394d79bbab2508a1372ad8253db146b961ca/base_report_to_printer/static/src/js/qweb_action_manager.js#L26), which cannot send ORM objects.

Please get the report by other means, like searching by name.

Reply:
Oh, didn't see that the method was called through XMLRPC.
So we need to fix the methods that passes a recordset as argument in the `ir.actions.report` model...